### PR TITLE
Feature/graph formatting

### PR DESCRIPTION
**Pull request by BDiehr**
Adding in New Graph Types and other small data changes.

### CODE REVIEW

File: modules/repository.js [hunk 1]
```diff
@@ -6,16 +6,18 @@ var repository = (function () {
...
-            genus: "G",
             // Meta Data
             dataType: "number",             // dataType is either "number" or "string"
             dataName: "hp",                 // name used to track the variable
             displayName: "Health Points",   // Appears on the Y-axis (the range) TODO: Edit display name in dashboard
             displayType: "scatter-plot",    // The display type for the dashboard, note that not every display type will be valid for every data type. TODO: Swap out string for some sort of javascript enum
+            // Categorical Fields
+            // primaryKey = kingdom + phylum + genus + dataName (i.e. "KPGhp" for this particular dataset
+            category: {
+                kingdom: "K",                   // Note: Assuming that categorical fields are Nullable
```
Comment (mdiehr) [inline modules/repository.js]: This assumption is safe, so you can remove the assumption

File: public/javascripts/flare.json [hunk 2]
```diff
@@ -0,0 +1,19 @@
+{
```
Comment (mdiehr) [inline public/javascripts/flare.json]: can you name this file "testdata" or something like that?

File: views/index.jade [hunk 3]
```diff
@@ -81,5 +83,14 @@ body#home2
     a(href='/login')
       | Sign up for free
     br
-    br
-    // TODO: Remove BR tags, get the bottom space in a smarter way - Brian
\ No newline at end of file
+    //  TODO: Move to Dashboard! Temporary Code For Testing Graph Functionality.
```
Comment (mdiehr) [inline views/index.jade]: enter an issue into github to track this todo

**Comment by mdiehr** [comment]
Looks good. Merge after addressing the comments.

**Comment by BDiehr** [comment]
Will do after work, thanks for the review!

**Comment by BDiehr** [comment]
Comments addressed in my last two commits. Let me know if there is anything else to change.

**Pull request merged**